python/kserve/kserve/protocol/dataplane.py

```diff
@@ -39,7 +39,8 @@
 # RayServeSyncHandle has been the return type of serve.run since Ray 2.5.
 # DeploymentHandle will be the new return type (still under feature flag in Ray 2.7).
 # ref https://github.com/ray-project/ray/pull/37817
-ModelHandleType = Union[Model, RayServeHandle, RayServeSyncHandle, DeploymentHandle]
+# On Ray 2.10, it now returns DeploymentHandle
```

Comment:
Can you link to a reference doc?

Reply:
added, will push the PR once CI finishes, to not re-trigger it

Reply:
done, thanks for the heads up @terrytangyuan